torchgeo/trainers/segmentation.py

```diff
@@ -80,6 +80,20 @@ def config_task(self) -> None:
                 f"Currently, supports 'ce', 'jaccard' or 'focal' loss."
             )
 
+        # Freeze backbone
+        if self.hyperparams.get("freeze_backbone", False) and self.hyperparams[
+            "model"
+        ] in ["unet", "deeplabv3+"]:
```

Comment:
What about fcn?

Reply:
FCN doesn't use a backbone.

Reply:
But you might still want to evaluate linear probing on it, right?